packages/react-composites/src/composites/CallWithChatComposite/ChatAndPeoplePane.tsx

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { concatStyleSets, DefaultButton, Stack } from '@fluentui/react';
+import { ChevronLeft28Regular } from '@fluentui/react-icons';
+import { ParticipantList, useTheme } from '@internal/react-components';
+import React, { useMemo } from 'react';
+import { CallAdapter } from '../CallComposite';
+import { CallAdapterProvider } from '../CallComposite/adapter/CallAdapterProvider';
+import { usePropsFor } from '../CallComposite/hooks/usePropsFor';
+import { ChatAdapter, ChatComposite } from '../ChatComposite';
+import { AvatarPersonaDataCallback } from '../common/AvatarPersona';
+import { ParticipantListWithHeading } from '../common/ParticipantContainer';
+import {
+  chatAndPeoplePaneButtonStyles,
+  chatAndPeoplePaneCloseButtonStyles,
+  chatAndPeoplePaneContentHiddenStyle,
+  chatAndPeoplePaneContentStyle,
+  chatAndPeoplePaneControlBarStyle,
+  chatAndPeoplePaneHiddenStyle,
+  chatAndPeoplePaneStyle
+} from './styles/ChatAndPeoplePaneStyles';
+import { useCallWithChatCompositeStrings } from './hooks/useCallWithChatCompositeStrings';
+import { removeParticipantFromCallWithChat } from './SidePane';
+
+export const ChatAndPeoplePane = (props: {
```

Comment:
I'm not a big fan of this combined component because it isn't cohesive.

My ideal design would be: A base `MobileEmbeddedPane` component that renders the back button, the chat/people switching buttons, and puts its children in a stack underneath.

Then, the existing `EmbeddedChatPane` and `EmbeddedPeoplePane` use the `MobileEmbeddedPane` if `mobileView` is set. That way `CallWithChatComposite` doesn't have the bit `mobileView` switch, and the chat vs people logic can stay separated as it is right now for desktop.

It does mean that both `EmbeddedChatPane` and `EmbeddedPeoplePane` will have to take an extra callback for switch to the other pane, which they don't need for desktop. But that's just one prop that's used only on mobile.

Reply:
# This is the big one.

Didn't look into the implementation here in detail. Let's resolve this first.

Reply:
Thanks for iterating on the code organization. Like the new approach a lot.